### PR TITLE
Test document.exitFullscreen() for an inactive document

### PR DESCRIPTION
**Pull request by foolip**
Test for https://github.com/whatwg/fullscreen/pull/68

### CODE REVIEW

File: fullscreen/api/document-exit-fullscreen-active-document.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<script>
+var t = async_test();
+
+onload = t.step_func(() => {
+  var iframe = document.querySelector("iframe");
+  var documentBeforeNav = iframe.contentDocument;
+
+  iframe.onload = t.step_func(() => {
+    var p = documentBeforeNav.exitFullscreen();
+    assert_true(p instanceof Promise, 'exitFullscreen() returns promise');
```
Comment (zcorpan) [inline fullscreen/api/document-exit-fullscreen-active-document.html]: The message is shown when the assert fails, and this message says the opposite of what happened when it fails. Maybe 'exitFullscreen() should return a promise'

**Review by zcorpan — APPROVED** [review]
LGTM % nit

File: fullscreen/api/document-exit-fullscreen-active-document.html (hunk 1 above)
Comment (foolip) [inline fullscreen/api/document-exit-fullscreen-active-document.html]: The failure message is "assert_true: exitFullscreen() returns promise expected true got false", isn't that appropriate? I've come to to just describe the expected value for assert_equals and the statement that is true of assert_true, based on what the errors end up looking like.

Comment (zcorpan) [inline fullscreen/api/document-exit-fullscreen-active-document.html]: Hmm I guess it is indeed clear. OK.

**Pull request merged**